client/flutter/lib/pages/onboarding/notifications_page.dart

```diff
@@ -54,7 +54,9 @@ class _NotificationsPageState extends State<NotificationsPage> {
   }
 
   void _registerFCMToken() async {
+    print("FCM Token registration called");
     await _firebaseMessaging.getToken().then((String token) {
+      UserPreferences().setFCMToken(token);
```

Comment:
It might be best to add `await` to this line and the next (to match the code in updateFCMToken in the other file)